public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,186 @@
+Thus the name of the newly created application 
+in the previous chapter needs to be passed as argument .(here it is rocky-hamlet-67506):
+
+```
+$ heroku git:remote -a rocky-hamlet-67506
+```
+
+Notice the remote endpoint is added to the git config
+
+```
+$ git remote -v
+heroku	https://git.heroku.com/rocky-hamlet-67506.git (fetch)
+heroku	https://git.heroku.com/rocky-hamlet-67506.git (push)
+```
+
+### Adding the buildpack
+
+Buildpacks are responsible for generate assets or compiled code.
+
+For more information browse the [Heroku documentation](https://devcenter.heroku.com/articles/buildpacks)
+
+For deployement the [Vibe.d buildpack](https://github.com/MartinNowak/heroku-buildpack-d) can be used : 
+
+```
+$ heroku buildpacks:set https://github.com/MartinNowak/heroku-buildpack-d
+```
+By default the buildpack uses the latest dmd compiler. 
+It is possible to use GDC or LDC and to choose a specific compiler versions by adding a `.d-compiler` file to your project. 
+
+Use `dmd`, `ldc`, or `gdc` to select the latest or dmd-2.068.2, ldc-0.16.0, or gdc-4.9.2 to 
+select a specific version of a compiler.
+
+### Deploy the code 
+
+Proceed in your usual git habit and write awesome code. 
+
+To release a new version, just push the newest version to the Heroku endpoint.
+
+```
+$ git add .
+$ git commit -am "My first vibe.d release"
+$ git push heroku master
+Counting objects: 9, done.
+Delta compression using up to 2 threads.
+Compressing objects: 100% (6/6), done.
+Writing objects: 100% (9/9), 997 bytes, done.
+Total 9 (delta 0), reused 0 (delta 0)
+
+-----> Fetching custom git buildpack... done
+-----> D (dub package manager) app detected
+-----> Building libevent
+-----> Building libev
+-----> Downloading DMD
+-----> Downloading dub package manager
+-----> Setting PATH:
+-----> Initializing toolchain
+-----> Building app
+       Running dub build ...
+Building configuration "application", build type release
+Running dmd (compile)...
+Compiling diet template 'index.dt' (compat)...
+Linking...
+       Build was successful
+-----> Discovering process types
+       Procfile declares types -> web
+-----> Compiled slug size: 3.5MB
+-----> Launching... done, v4
+       https://rocky-hamlet-67506.herokuapp.com/ deployed to Heroku
+To git@heroku.com:rocky-hamlet-67506.git
+ * [new branch]      master -> master
+```
+
+Open the app in the browser with the following command
+
+```
+$ heroku open
+```
+
+### Scaling dynos containers 
+
+After deploying, the app is running on a web dyno. 
+Think of a dyno as a lightweight container that runs the command specified in the Procfile.
+
+To check how many dynos are running using the ps command:
+
+```
+$ heroku ps
+Free dyno hours quota remaining this month: 550h 0m (100%)
+For more information on dyno sleeping and how to upgrade, see:
+https://devcenter.heroku.com/articles/dyno-sleeping
+
+No dynos on ⬢ rocky-hamlet-67506
+```
+
+By default, the app is deployed on a free dyno which doesn't access request. 
```

Comment:
It mean that as the app is deployed on a free dyno, it can't receive requests you have to manually "open" the dyno to access request.
It's the normal way to use heroku free. For more information refer to the [documentation](https://devcenter.heroku.com/articles/scaling)